src/CoreWCF.Queue/src/CoreWCF/Queue/Common/QueueProcessMessage.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using CoreWCF.Queue.Common.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CoreWCF.Queue.Common
+{
+    public class QueueProcessMessage
+    {
```

Comment:
Make this internal. Anything we make public has it's api locked down which doesn't allow for evolving it as easily if we need to make changes.